packages/neo-one-server-plugin/src/version.ts

```diff
@@ -1,2 +1,5 @@
+import * as fs from 'fs-extra';
+import path from 'path';
```

Comment:
Use `import * as path from 'path';` to be consistent with how we import it elsewhere.

Reply:
👍